common/graph_db_client.py

```diff
@@ -87,72 +88,79 @@ def element_exists(self,element_type:str,element_id:int,conditions:str="")->bool
-                    ".property('description',prop_description)"
-                    ".property('id',prop_id)"
-                    ".property('human_readable_id',prop_human_readable_id)"
-                    ".property('source_degree',prop_source_degree)"
-                    ".property('target_degree',prop_target_degree)"
-                    ".property('rank',prop_rank)"
-                    ".property('source',prop_source)"
-                    ".property('target',prop_target)"
-                ),
-                bindings={
-                    "prop_partition_key": "entities",
-                    "prop_source_id": row.source,
-                    "prop_target_id": row.target,
-                    "prop_weight": row.weight,
-                    "prop_text_unit_ids":json.dumps(row.text_unit_ids.tolist() if row.text_unit_ids is not None else []),
-                    "prop_description": row.description,
-                    "prop_id": row.id,
-                    "prop_human_readable_id": row.human_readable_id,
-                    "prop_source_degree": row.source_degree,
-                    "prop_target_degree": row.target_degree,
-                    "prop_rank": row.rank,
-                    "prop_source": row.source,
-                    "prop_target": row.target,
-                },
+                    ".to(g.V().has('name',prop_target_id"+str(iter_row)+"))"
+                    ".property('weight',prop_weight"+str(iter_row)+")"
+                    ".property('description',prop_description"+str(iter_row)+")"
+                    ".property('id',prop_id"+str(iter_row)+")"
+                    ".property('human_readable_id',prop_human_readable_id"+str(iter_row)+")"
+                    ".property('source_degree',prop_source_degree"+str(iter_row)+")"
+                    ".property('target_degree',prop_target_degree"+str(iter_row)+")"
+                    ".property('rank',prop_rank"+str(iter_row)+")"
+                    ".property('source',prop_source"+str(iter_row)+")"
+                    ".property('target',prop_target"+str(iter_row)+")"
+                    ".property(list,'text_unit_ids',prop_text_unit_ids"+str(iter_row)+")"
+                )
+                query_bindings.update({
+                    ("prop_source_id"+str(iter_row)): row.source,
+                    ("prop_target_id"+str(iter_row)): row.target,
+                    ("prop_weight"+str(iter_row)): row.weight,
+                    ("prop_description"+str(iter_row)): row.description,
+                    ("prop_id"+str(iter_row)): row.id,
+                    ("prop_human_readable_id"+str(iter_row)): row.human_readable_id,
+                    ("prop_source_degree"+str(iter_row)): row.source_degree,
+                    ("prop_target_degree"+str(iter_row)): row.target_degree,
+                    ("prop_rank"+str(iter_row)): row.rank,
+                    ("prop_source"+str(iter_row)): row.source,
```

Comment:
```suggestion
```
You don't need row.source row.target again. You already have that above. I believe you only need:
source_id (not name but entity_id of the source)
target_id
weight
text_units
id
maybe description? It would be nice to have since it is a AI generated summary of the relationship/connection, but I wouldn't be surprised if that is text and thus not desired in the graphrag.

Reply:
We can get rid of description as well as rank and degree since they can be computed directly, we would just need to calculate them when we query